packages/renderer/src/lib/chat/components/chat.svelte

```diff
@@ -1,16 +1,19 @@
 <script lang="ts">
-import { Chat, type UIMessage } from '@ai-sdk/svelte';
+import { Chat } from '@ai-sdk/svelte';
 import type { Attachment } from '@ai-sdk/ui-utils';
 import { untrack } from 'svelte';
 import { toast } from 'svelte-sonner';
 
 import type { ModelInfo } from '/@/lib/chat/components/model-info';
 import { ChatHistory } from '/@/lib/chat/hooks/chat-history.svelte';
+import { convertToUIMessages } from '/@/lib/chat/utils/chat';
 import { getModels } from '/@/lib/models/models-utils';
+import { mcpRemoteServerInfos } from '/@/stores/mcp-remote-servers';
 import { providerInfos } from '/@/stores/providers';
+import { MessageConfigSchema } from '/@api/chat/message-config';
 import type { MCPRemoteServerInfo } from '/@api/mcp/mcp-server-info';
 
-import type { Chat as DbChat } from '../../../../../main/src/chat/db/schema';
+import type { Chat as DbChat, Message as DbMessage } from '../../../../../main/src/chat/db/schema';
```

Comment:
I guess we should move it to api package at some point ? (b/c here renderer uses main stuff which should be prohibited)

Reply:
I agree, I can do a proposition in a follow up PR, thanks for the approval.

Reply:
https://github.com/kortex-hub/kortex/pull/467